File: program23.java
import java.util.Scanner;

public class program23
{
    
    public static void main(String arg[])
    {
        Scanner reader = new Scanner(System.in);

        System.out.print("Enter the number:");
        int num = reader.nextInt();

        if(num % 2 == 0)
            System.out.println(num + "is even");
        else
            System.out.println(num + "is odd");
    }
}

File: program23.c
// ACCEPT NUMBER FROM USER AND CHECK WHETHER IT IS EVEN OR ODDD
#include<stdio.h>
#include<stdbool.h>

bool CheckEven(int iNo)
{
    if((iNo % 2) == 0)
    { 
        return true;
    }
    else 
    {
        return false;

    }

}
int main()
{
    int iValue = 0;
    bool bRet = false;
    
    printf("enter number\n");
    scanf("%d", &iValue);

    bRet = CheckEven(iValue);
    if(bRet == true)
    {
        printf("%d is even number", iValue);
    }
    else
    {
        printf("%d is odd number", iValue);
    }
    return 0;

}
